mirrormanager2/app.py

```diff
@@ -105,7 +105,7 @@ def create_app(config=None):
     # Admin UI
     # Flask-Admin does not support having a single instance and multiple calls
     # to init_app() (it stores the app as an instance attribute)
-    ADMIN = Admin(template_mode="bootstrap3")
+    ADMIN = Admin(template_mode="bootstrap4")
```

Comment:
flask-admin doesnt have a bootstrap5 template mode, so we use the bootstrap4 one... it works ok enough for the admin pages